apps/docs/src/examples/inputs/TagsSuggestions.tsx

```diff
@@ -0,0 +1,114 @@
+import { useEffect, useRef, useState } from "react";
+import { Popper } from "@mui/material";
+import {
+  HvLabel,
+  HvPanel,
+  HvTag,
+  HvTagsInput,
+  HvTypography,
+} from "@hitachivantara/uikit-react-core";
+import { DropDownXS } from "@hitachivantara/uikit-react-icons";
+
+const colors = [
+  "Blue",
+  "Red",
+  "Green",
+  "Yellow",
+  "Purple",
+  "White",
+  "Black",
+  "Orange",
+  "Pink",
+  "Brown",
+  "Gray",
+  "Cyan",
+  "Magenta",
+  "Lime",
+  "Teal",
+  "Lavender",
+];
+
+const lastUsed = ["Blue", "Red", "Green"];
+
+export default function Demo() {
+  const containerRef = useRef<HTMLDivElement>(null);
+  const [selectedColors, setSelectedColors] = useState<string[]>([]);
+  const [open, setOpen] = useState(false);
+
+  useEffect(() => {
+    if (containerRef.current) {
+      containerRef.current.scrollLeft = containerRef.current.scrollWidth;
+    }
+  }, [selectedColors]);
+
+  const handleAddColor = (color: string) => {
+    setSelectedColors((prev) => {
+      const newColors = prev.filter((c) => c !== color);
+      return newColors.length ? [...newColors, color] : [color];
+    });
+  };
+
+  const handleRemoveColor = (color: string) => {
+    setSelectedColors((prev) => prev.filter((c) => c !== color));
+  };
+
+  return (
+    <div className="w-300px">
+      <HvLabel>Tags with suggestions</HvLabel>
+      <HvTagsInput
+        ref={containerRef}
+        onChange={(event, value) => {
+          setSelectedColors(
+            value.map((v) => (typeof v === "string" ? v : (v.label as string))),
+          );
+        }}
+        onDelete={(_, value) => {
+          handleRemoveColor(value as string);
+        }}
+        value={selectedColors}
+        commitTagOn={["Comma"]}
+        endAdornment={
+          <DropDownXS
+            rotate={open}
+            onClick={() => setOpen((o) => !o)}
+            className="cursor-pointer"
+            size="xs"
+          />
+        }
+        classes={{
+          tagsList: "h-32px pr-0 overflow-hidden",
+        }}
+      />
+      <Popper anchorEl={containerRef.current} open={open}>
+        <HvPanel className="flex flex-wrap flex-col gap-xs w-300px bg-bgContainer border-rounded-16px border border-atmo3">
```

Comment:
maybe there's a better middle ground, but the `HvPanel` defaults currently look better IMO. `HvPanel` already has the correct `backgroundColor`, and a better `borderColor`
```suggestion
        <HvPanel className="flex flex-wrap flex-col gap-xs w-300px border">
```

![image](https://github.com/user-attachments/assets/613444a5-9487-4d07-bdff-eb533516f8ff) 
`HvPanel` default:
![image](https://github.com/user-attachments/assets/e8130f7d-de64-4b46-8ee5-acf6197a3f0f)